src/app/[locale]/asset/_components/RedirectToViewer.tsx

```diff
@@ -33,9 +36,12 @@ export const RedirectToViewer = () => {
     }, []);
 
     async function navigateToViewerOfAsset(assetId: string | undefined): Promise<void> {
-        const aasIds = await getAasIdsOfAsset(assetId);
-        assertAnAasIdExists(aasIds);
-        const targetUrl = determineViewerTargetUrl(aasIds);
+        const { isSuccess, result } = await getAasIdsOfAsset(assetId);
```

Comment:
i'd like it if we made more explicit what "result" is, i.e., const {iSuccess, result: aasIds} = await getAasIdsOfAsset(...)

Reply:
Ok